cvmfs/cvmfs_config

```diff
@@ -1256,7 +1264,11 @@ cvmfs_probe() {
         cvmfs_readconfig $fqrn || die "Failed to read CernVM-FS configuration"
         echo -n "Probing $CVMFS_MOUNT_DIR/$fqrn... "
         ls "$CVMFS_MOUNT_DIR/$fqrn" > /dev/null 2>&1  # workaround for FC28+
-        df -P "$CVMFS_MOUNT_DIR/$fqrn" 2>&1 | grep -q ^cvmfs2
+        if [ "$sys_arch" = "Darwin" ] && ! is_macfuse_enabled; then
+          df -P "$CVMFS_MOUNT_DIR/$fqrn" 2>&1 | grep -q ^fuse-t
```

Comment:
It seems like this df output is not always consistent with fuse-t ( depending on how you install it ), this may be more robust
```suggestion
          df -P "$CVMFS_MOUNT_DIR/$fqrn" 2>&1 | grep -q $fqrn
```

Reply:
Done